README.rdoc

```diff
@@ -29,6 +29,49 @@ Converting a Magellan track log to GPX:
          GPX::MagellanTrackLog::convert_to_gpx(filename, "#{filename}.gpx")
       end
 
+Exporting an ActiveRecord to GPXFile (as Waypoints)
+
+    #
```

Comment:
Would you mind wrapping this code sample in a [Ruby GFM code fence](https://help.github.com/articles/github-flavored-markdown/#syntax-highlighting)? That way we can have syntax highlighting on the following lines.